realm/realm-library/src/androidTest/kotlin/io/realm/entities/embedded/EmbeddedCircularParentWithoutPrimaryKey.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.entities.embedded
+
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+import java.util.*
+
+// Parent pointing to an embedded object that has a circular schema, i.e. objects can point
+// to themselves. Note, this isn't actually allowed at runtime. Only at schema validation time.
+open class EmbeddedCircularParentWithoutPrimaryKey(@PrimaryKey var id: String = UUID.randomUUID().toString()) : RealmObject() {
```

Comment:
Is this correct? Class name is `EmbeddedCircularParentWithoutPrimaryKey` and the first field has `@PrimaryKey`.

Reply:
No. Caught it myself, so already updated.